projects/components/src/datagrid/interfaces/datagrid-action-display.interface.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+import {
+    ActionStyling,
+    ContextualActionDropdownDisplayConfig,
+    ContextualActionInlineDisplayConfig,
+    TextIcon,
+} from '../../common/interfaces';
+import { ContextualActionPosition } from '../datagrid.component';
+
+/**
+ * Makes the featured count as required when action menu is displayed as a dropdown and also adds the position
+ * at which the actions have to displayed in a datagrid
+ */
+interface DatagridContextualActionDropdownDisplayConfig extends ContextualActionDropdownDisplayConfig {
```

Comment:
What I don't get is: how position in a row will handle `featuredCount`

Reply:
Position in a row does not do anything different about the featured count. The number of featured actions shown is the same whether displayed in a row or outside